openmetadata-ui/src/main/resources/ui/src/components/AlertBar/AlertBar.interface.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+interface AlertBarProps {
+  type: 'general' | 'info' | 'success' | 'warning' | 'error';
```

Comment:
Create one union type AlertBarType and assign that type her.

Reply:
Updated.